PowerValidatedSolutions.psm1

```diff
@@ -22159,7 +22159,7 @@ Function Set-NsxtApplianceUserPassword {
 }
 Export-ModuleMember -Function Set-NsxtApplianceUserPassword
 
-#EndRegion  End NSX-T Functions                                ######
+#EndRegion  End NSX Functions                                ######
```

Comment:
Needs spaces added to align hashes

Reply:
Addressed in https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/pull/252/commits/b858c50d0a5573c5284a93d4384b3db0d4ece27c.